microsite/data/plugins/ibm-apic-backend.yaml

```diff
@@ -0,0 +1,16 @@
+---
+title: IBM APIC Backend
+author: croz.net
+authorUrl: https://croz.net/?utm_source=backstage.io&utm_medium=marketplace&utm_campaign=backstage-ibm-apic-backend
+category: API Management
+description: Bring IBM APIC to Backstage.
+documentation: https://croz.net/news/backstage-plugin-ibm-apic?utm_source=backstage.io&utm_medium=marketplace&utm_campaign=backstage-ibm-apic-backend
+iconUrl: https://croz.net/wp-content/uploads/2024/02/croz_large-o.png
+npmPackageName: '@croz/plugin-ibm-apic-backend'
+tags:
+  - idp
+  - developer
+  - portal
+  - internal
```

Comment:
It's on a dedicated Backstage directory so it's all of these aren't needed - idp, developer, portal, internal.

Reply:
I've just copied the example from the docs. Will remove those.

Reply:
OK, can you point me to the docs you used? This is where I generally send people but I don't see it there - https://backstage.io/docs/plugins/add-to-directory 🤔

Reply:
Oh :) My bad. I saw it in the existing plugins eq. https://github.com/backstage/backstage/blob/master/microsite/data/plugins/argo-cd.yaml.
I'll remove those.

Reply:
Ok, that's fair then. We should probably remove them at some point. The tags actually aren't used for anything right now just the category.